Src/Add-AzureDtlVM.ps1

```diff
@@ -1,64 +1,142 @@
+<#
+.SYNOPSIS 
+    This script adds the specified number of Azure virtual machines in the specified DevTest Lab.
```

Comment:
Sorry, I meant 'in all the rest of the source files whenever you use this phrase'